docs/dask_cudf/source/index.rst

```diff
@@ -6,8 +6,9 @@
 Welcome to dask-cudf's documentation!
 =====================================
 
-Dask-cuDF is an extension library for the `Dask <https://dask.org>`__
-parallel computing framework that provides a `cuDF
+**Dask-cuDF** (pronounced "desk-cuh-dee-ef") is an extension
```

Comment:
I would skip the pronunciation guide on the dask-cudf library. I think we can just do it for the main cuDF README. Also dask isn’t the same as desk in my pronunciation but I’m not enough of an expert to know how to write that word pronunciation properly.

Reply:
> I would skip the pronunciation guide on the dask-cudf library. I think we can just do it for the main cuDF README.

I wasn't sure if we should do it here either but decided in favor of this for the cases where people may end up directly here when looking to accelerate dataframes with Dask on the GPU.

> Also dask isn’t the same as desk in my pronunciation but I’m not enough of an expert to know how to write that word pronunciation properly.

From community meetings I've joined, "desk" is definitely more prevalent.

Reply:
...Doesn't dask rhyme with task or mask, and not desk (or the first syllable of "pesky")? I think that dask is a self-explanatory pronunciation and I don't see a way to explain the pronunciation better than the word "dask" itself.

Reply:
I gotta be honest and I always thought "mask", "task" and "desk" sounded the same (with the exception of the first letter, of course), then I went on to listen to them and can now grasp the (to me, subtle) difference. I agree with you that "dask" sounds like "dask" and not "desk".

With that said I'd say we can update here and say it's pronounced "dask-KOO-dee-eff". Once again, my primary purpose is to make sure people who land here as opposed to looking at cuDF by itself also know how to pronounce cuDF, Dask pronunciation was added just for completeness. If you still think we should remove it I can do that, but personally I feel there's a benefit to have it here too.